### ISSUE TITLE
Rename "tools" module

### ISSUE THREAD

**Issue by miniche** [CentralReport/CentralReport#26]
In CentralReport, we have the "tools" module. It contains the... "Config" class.

We could find a better name for this module, more representative of his content.
Do you have any idea?

**Comment by Guimove** [comment]
Are you talking about de "tools" folder that contain the install/uninstall scripts ?
If yes, we should rename it like : Init, Config, Install...something like that.

**Comment by miniche** [comment]
No, I'm talking about the "tools.py" module in cr package (This module contains the Config python class).

**Comment by Ninir** [comment]
Will this module contain something else than the config class?

If not, let's keep things simple: if this is related to the config class, it should be named config.

I'm used to have one class for one purpose, meaning... "Single Responsibility Principle". 
(However, as you can see packages in Python, we should find a way to organize things. Task not directly related to this issue.)

**Comment by miniche** [comment]
In the future, I don't know.
In python, each python file is a module and represent one or more classes. If we rename "tools" in "config", we must use "cr.config.Config()".
I'll try to find some examples on Internet.

**Comment by miniche** [comment]
This module will be refactored in a next release, perhaps when we will implement plugin support.

This issue is paused.
